Framework/src/Triggers.cxx

```diff
@@ -68,8 +68,25 @@ std::string createKafkaGroupId(std::string_view prefix, std::string_view detecto
   return groupId;
 }
 
+bool checkKafkaParams(const std::string& kafkaBrokers, const std::string& topic, const std::string_view triggerTypeLogId)
+{
```

Comment:
genuine question: why don't you use a string_view for the first two arguments ? 
I understand that the parameters you will give later in SOR are strings, but from the point of view of the interface, wouldn't it be better to have something uniform ?

Reply:
you are right, all of these should be the same. Thanks for the remark
I generally prefer `std::string_view` over `const std::string&` because it is non owning and non allocating and read only class, so it 1) implies the usage, 2) allows you to use the function (generally speaking) allocating something/creating temporary string, 3) string_view can be used easily with wider range of classes: `std::string`, `const char*`, `std::vector<char>`, ... But in this case the choice does not really matter...

Reply:
ok !